### PR TITLE
Added enabling/disabling prevYear and nextYear buttons

### PR DESCRIPTION
**Pull request by paw3lx**
Related to #3732 

I added disabling/enabling previous and next year buttons, depending on validRange. I think they should work in the same way as prev/next buttons.

The other problem with disabling next button is already solved.

### CODE REVIEW

File: src/Calendar.toolbar.js [hunk 1]
```diff
@@ -87,6 +87,20 @@ Calendar.mixin({
 				'disableButton',
 			'next'
 		);
+
+		this.toolbarsManager.proxyCall(
+			prevInfo.isValid ?
```
Comment (arshaw) [inline src/Calendar.toolbar.js]: the `prevInfo` and `nextInfo` objects you are leveraging here are information for the prev and next click (for example, one month ahead or back), not necessarily one year ahead or one year back. probably fine in 99% of usecases, but would prefer if you generated and tested a date profile for one year forward/back explicitly.

Comment (paw3lx) [inline src/Calendar.toolbar.js]: Ok, I understand. But in my opinion the date profile for one year forward will be the same as prev date profile builded from buildPrevDateProfile method, right?
Or maybe I don't understand when prev and next year buttons should be enabled/disabled. I think that prevYear button should be enabled when you can go back at least one day, right? Similarly with nextYear.

Comment (arshaw) [reply src/Calendar.toolbar.js]: @paw3lx that would be a convenient shortcut, but no, sometimes the user could go back a day but could NOT go back a year due to the [validRange](https://fullcalendar.io/docs/current_date/validRange/) limiting that large of a jump in navigation

**Comment by arshaw** [comment]
I'm closing this PR because it has gotten way out of sync with the main codebase (my fault), but I created this new issue to track demand for it: https://github.com/fullcalendar/fullcalendar/issues/5575

**Pull request closed without merging**